### ISSUE TITLE
Helper that outputs an Inno Setup file for an application

### ISSUE THREAD

**Issue by trevordevore** [trevordevore/levure#43]
Inno setup is free and has a text based installer format. If a signing tool is configured in the Inno setup installer than building an installer will automatically sign the executable.

**Comment by trevordevore** [comment]
I've been doing some work on this on the following branch:

https://github.com/trevordevore/levure/tree/tkd-packaging-callbacks

This branch allows helpers to receive callbacks during the packaging process as well as define new keys for the app.yml file that can affect packaging.

The into_setup helper repo is at the following url. The README has instructions.

https://github.com/trevordevore/levurehelper-inno-setup

**Comment by trevordevore** [comment]
There is also a branch for DropDMG on macOS:

https://github.com/trevordevore/levurehelper-dropDMG

**Comment by macMikey** [comment]
Cool.  I was about to start poking around in the installer goodies.

**Comment by trevordevore** [comment]
I've merged in changes into `master` branch and released version to `0.9.1`.

I've added a doc to the wiki explaining how to build installers using helpers.

https://github.com/trevordevore/levure/wiki/Building-Installers

I've added docs explaining `registered components` and `packager callbacks stockpile` properties in the `helper.yml` file. These two features allow installers to be created (among other things).

https://github.com/trevordevore/levure/wiki/helpers
